Makefile

```diff
@@ -528,6 +528,9 @@ prepare-release: ## Generates a makefile that will override environment variable
 	$(MAKE) bundle
 	$(MAKE) helm-build VERSION=$(VERSION)
 
+.PHONY: read-release-version
+read-release-version: ## Reads release version
+	@echo "v$(VERSION)"
```

Comment:
:+1: Using the single version from the makefile makes sense to me. On release branches we set this as an override as part of the release process https://github.com/Kuadrant/dns-operator/blob/v0.6.0/make/release.mk#L7

Reply:
I know :) 

I think that I will bring this release.mk approach to the other projects